configs/train/tiny.yaml

```diff
@@ -1,7 +1,3 @@
-insight:
-  enabled: false
-  evaluate_every: 500
-  peak: 65535
 model:

```

Comment:
Let's call this `train_example_config_small.yaml`.  Also, please include the yaml config for the best CellMincer model from the paper and call it `train_example_config_default.yaml`.